.circleci/run-stackrox-ci/10-run-stackrox-ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+BRANCH=$1
+BUILD_NUM=$2
+
+comment_on_pr_with_ci_link() {
+    local ci_url=$1
+
+    wget --quiet https://github.com/joshdk/hub-comment/releases/download/0.1.0-rc6/hub-comment_linux_amd64
+    sudo install hub-comment_linux_amd64 /usr/bin/hub-comment
+
+    export CI_URL=$ci_url
+
+    hub-comment -template-file "${CI_ROOT}/run-stackrox-ci/ci-url-comment-template.tpl"
+}
+
+git clone git@github.com:stackrox/stackrox.git
+cd stackrox
+
+git config user.email "$GITHUB_USER_NAME"
+git config user.name "$GITHUB_USER_EMAIL"
+
+stackrox_branch="${BRANCH}_ROBOT_COLLECTOR_CI"
```

Comment:
[ultra-nit] This is completely optional, but maybe we can group all collector CI branches by prepending `collector_ci/` instead? That way the commands for listing or removing these branches will use the pattern `collector_ci/*` which I think makes it a little clearer on the intent.
```suggestion
stackrox_branch="collector_ci/${BRANCH}"
```

Reply:
Done